noxfile.py

```diff
@@ -407,13 +407,36 @@ def check_changelog(session: nox.Session):
 
 @nox.session(name="set-minimal-package-versions")
```

Comment:
Somewhat unrelated, but I noticed it from the log you pasted here, shouldn't this session use `venv_backend="none"` as well since we don't need the venv here?